### PR TITLE
Revert "ci: temporary disable aarch64 packaging job"

### PR DESCRIPTION
**Pull request by ylobankov**
This reverts commit ef3152cdc249045be8250c65d47b1a002e49467e.

We have solved the issues with aarch64 runners, so we can enable back aarch64 jobs.

### CODE REVIEW

**Comment by sergos** [comment]
All 3 failing tests passed on the `sergos` worker.

**Review by ochaplashkin — APPROVED** [review]

**Review by Totktonada — APPROVED** [review]

File: .github/actions/environment/action.yml [hunk 1]
```diff
@@ -5,12 +5,13 @@ runs:
   steps:
     - run: |
         echo VARDIR=/tmp/t | tee -a $GITHUB_ENV
+        echo TEST_RUN_JOBS=$(nproc || sysctl -n hw.logicalcpu || sysctl -n hw.ncpu) | tee -a $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (Totktonada) [reply .github/actions/environment/action.yml:8]: I think it requires a little clarification in the commit message.

AFAIU, there is a problem on the new aarch64 runner machines with how test-run calculates a number of CPUs (python's `multiprocessing.cpu_count()` call). I would like to know what it gives and what `nproc`/`sysctl` gives on these machines.

And there is another change: we replace `jobs = 2 * CPU count` with `jobs = CPU count`. This is logically different change and I would like to have it mentioned in the commit message.

Feel free to ignore. It is just for the sake of completeness.

[the remark this answers is not in the document]
Comment (ylobankov) [reply .github/actions/environment/action.yml:8]: Ok, looks like you are right. Let's make the change for the number of test-run workers as a separate one.

[the remark this answers is not in the document]
Comment (Totktonada) [reply .github/actions/environment/action.yml]: I don't mind to make these changes in one commit. I just want them to be described properly:

> There is a problem with docker inside LDX, bla-bla. It is fixed by TEST_RUN_JOBS.
>
> While we're on it, another problem is fixed: test-run's default `CPUs * 2` heuristic works bad on aarch64 machines, bla-bla. Changed to just number of CPUs.

Something of this kind.

[the remark this answers is not in the document]
Comment (ylobankov) [reply .github/actions/environment/action.yml]: I will leave the value as it was before. It's better to dedicate a separate PR for it.

[the remark this answers is not in the document]
Comment (Totktonada) [reply .github/actions/environment/action.yml]: @ylobankov I've looked around based on your information and created a change in test-run: https://github.com/tarantool/test-run/pull/435

[the remark this answers is not in the document]
Comment (ylobankov) [reply .github/actions/environment/action.yml]: Ok, let's bump it, thanks.

**Comment by ylobankov** [comment]
Cherry picked to 3.1 as well.

**Pull request merged**